File: Mobile/LetsGo/lib/models/distanciaRestaurante.dart
class DistanciaRestaurante {

  String distancia;
  int distanciaEmMetros;

  DistanciaRestaurante({this.distancia, this.distanciaEmMetros});

  factory DistanciaRestaurante.fromJson(Map<String, dynamic> json) {
    return DistanciaRestaurante(
      distancia: json['distancia'],
      distanciaEmMetros: json['distanciaEmMetros']
    );
  }
}

File: LetsGo/Domain/LetsGo.Domain/Entities/DistanciaRestaurante.cs
using System;
using System.Collections.Generic;

namespace LetsGo.Domain.Entities
{
    public class DistanciaRestaurante
    {
        public Restaurante Restaurante { get; set; }
        public string Distancia { get; set; }
        public int DistanciaEmMetros { get; set; }
    }
}